unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraphQueries.cs

```diff
@@ -1,25 +1,28 @@
 public static class TheGraphQueries
 {
-    public static readonly string getLandQuery = @"
+    private const int MAX_LAND_QUERY_AMOUNT = 1000;
+    public static string getLandQuery => landQuery.Replace("[AMOUNT]", MAX_LAND_QUERY_AMOUNT.ToString());
+
+    private static readonly string landQuery = @"
   query Land($address: Bytes) {
-    ownerParcels: parcels(first: 500, where: { estate: null, owner: $address }) {
+    ownerParcels: parcels(first: [AMOUNT], where: { estate: null, owner: $address }) {
       ...parcelFields
     }
-    ownerEstates: estates(first: 500, where: { owner: $address }) {
+    ownerEstates: estates(first: [AMOUNT], where: { owner: $address }) {
```

Comment:
Could we use a variable here like `$address` for amount instead of using a string replace?

Reply:
+1

Reply:
Sure, but the problem is that $address is replaced in the TheGraph part of the query ( not on our side) so if in the future someone adds a variable $amount for the query in the TheGraph part it will collide!

Reply:
We are actually sending the query and it's variables. It's not something happening in black box on TheGraph side.
Here we are setting `$address` variable for example
https://github.com/decentraland/unity-renderer/blob/master/unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs#L87

Reply:
I think that what you mean is to replace the value in thegraph, I have change it!

Reply:
Didn't see the comment but I realize after a while. Yep, I have changed it!

Reply:
Great :)
Don't forget you must also change the query args
`query Land($address: Bytes)`  to something like `query Land($address: Bytes, $amount: Int)`
and also maybe create a new variable class since `AddressVariable` is used for other stuff that doesn't require `$amount`

Reply:
I just reverted the changes,  to let them as they were before my changes, their API is no accepting an array of variables and the query needs to be changed since the fragment also needs the variable. I think that invest more hours into this to change that is not worth it